src/Serilog.Sinks.OpenTelemetry/OpenTelemetryLoggerConfigurationExtensions.cs

```diff
@@ -80,27 +83,32 @@ public static class OpenTelemetryLoggerConfigurationExtensions
         LoggingLevelSwitch? levelSwitch = null,
         int batchSizeLimit = 100,
         int batchPeriod = 2,
-        int batchQueueLimit = 10000)
+        int batchQueueLimit = 10000,
+        bool disableBatching = false)
     {
         if (sinkConfiguration == null) throw new ArgumentNullException(nameof(sinkConfiguration));
 
-        var sink = new OpenTelemetrySink(
+        var openTelemetrySink = new OpenTelemetrySink(
             endpoint: endpoint,
             protocol: protocol,
             formatProvider: formatProvider,
             resourceAttributes: resourceAttributes,
             headers: headers);
 
-        var batchingOptions = new PeriodicBatchingSinkOptions
+        ILogEventSink sink = openTelemetrySink;
+        if (disableBatching == false)
```

Comment:
Can this be changed to `!disableBatching` rather than `disableBatching == false`?